manifests/init.pp

```diff
@@ -1,34 +1,23 @@
 # Main file for puppet-ssh
 class ssh (
-  $server_options         = {},
-  $server_match_block     = {},
-  $client_options         = {},
-  $users_client_options   = {},
-  $version                = 'present',
-  $storeconfigs_enabled   = true,
-  $validate_sshd_file     = $::ssh::params::validate_sshd_file,
-  $use_augeas            = false,
-  $server_options_absent = [],
-  $client_options_absent = [],
-  $use_issue_net         = false,
+  Hash $server_options = {},
```

Comment:
@pmuller this might be my personal preference or my own interpretation of the style guide but it would be nicer if all the "$" were aligned.
Eg:
```
Hash     $server_match_block = {},
Boolean  $storeconfigs_enabled = true,
```